.gitattributes

```diff
@@ -6,3 +6,4 @@
 *.glb filter=lfs diff=lfs merge=lfs -text
 *.glc filter=lfs diff=lfs merge=lfs -text
 *.glm filter=lfs diff=lfs merge=lfs -text
+earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/test_data/ binary
```

Comment:
This was really difficult to test because after resetting my global setting back to autocrlf to true (the default), every time I would clone the repo fresh it would, of course checkout master. That means it would have the version with dos format files. But, switching branches to the one with the updated .gitattributes alone will not convert the line endings (very frustating). It turns out you have to do this in order to convert the line endings:

1. git checkout I19-Portable-build-Windows-fixes
1. rm .git/index from top level of the repo
1. git reset (rebuilds the index, but takes a few minutes)
1. git reset --hard HEAD

If it converts the line endings of the files it will show the files as being modified during git reset and if you do a git status before running git reset -- hard HEAD.

Anyway, long story short, I was able to verify this works, but I worry that users who want to fix this issue if it's encountered need some instructions on how to fix it even with the addition of the .gitattributes. It should work fine for new clones, but won't fix the issue for existing clones without some sort of manual update. Should we add a FAQ entry, maybe?